### PR TITLE
⚡feat: support loading from .env

### PR DESCRIPTION
**Pull request by Yedidyar**

### CODE REVIEW

**Comment by Yedidyar** [comment]
implemtes the last task #1

**Review by jycouet — APPROVED** [review]

**Review by jycouet — APPROVED** [review]
LGTM

File: src/cli.ts [hunk 1]
```diff
@@ -1,32 +1,96 @@
...
-				demandOption: true,
+				default: process.env["DATABASE_URL"],
 				description:
 					"Your PostgreSQL database connection string. Only PostgreSQL databases are supported.",
 			},
 			output: {
-				default: "./src/shared",
+				default: process.env["OUTPUT"] ?? "./src/shared",
+			},
+			tableProps: {
```
Comment (Yedidyar) [inline src/cli.ts:34]: Maybe should it be extendable we can use clark Promt for that

File: src/cli.ts [hunk 2]
```diff
@@ -1,32 +1,96 @@
...
+	spinner.start("Generating everything for you");
+	const report = await getEntitiesTypescriptPostgres(
+		connectionString,
+		output,
+		tableProps,
+		tmp_jyc
+	);
+	spinner.stop(`Generation done ${green(`✓`)}`);
+
+	logReport("full", report);
```
Comment (Yedidyar) [inline src/cli.ts:89]: Maybe should be an option with default

Comment (Yedidyar) [reply src/cli.ts:89]: I created issue

File: src/cli.ts (hunk 1 above)
Comment (Yedidyar) [reply src/cli.ts:34]: For now lets leavel it

**Pull request merged**